src/themes/shared/borders.js

```diff
@@ -9,34 +9,88 @@ import type {BordersT} from '../types.js';
 
 const borders: BordersT = {
   border100: {
-    borderColor: 'hsla(0, 0%, 0%, 0.04)',
-    borderStyle: 'solid',
-    borderWidth: '1px',
+    borderTopColor: 'hsla(0, 0%, 0%, 0.04)',
```

Comment:
I think its usage should be changed, not the theme object. Additionally, it would be a breaking change.

Reply:
I wasn’t happy with this either. Updated to change only the styled components.